### PR TITLE
feat: new circleci guide (DOC-7895)

### PR DESCRIPTION
**Pull request by jcountsNR**
New guide for circleci/new relic integration.

IMPORTANT: I need help from the dev team to decide where this lives both in the repo and in the website as a whole.

A lot of the template I used came from the existing [cloudflare doc](https://docs.newrelic.com/docs/logs/forward-logs/cloudflare-logpush-forwarding/) since this is a very similar page, only for CircleCI. 

The guide covers 2 main areas:
1. how to set up a circleCI webhook to send logs to new relic
2. what you can do in New Relic with those logs (mainly the quickstart already built)

### CODE REVIEW

**Comment by paperclypse** [comment]
The person assigned to this Jira work will review and update this draft.

File: src/content/docs/logs/forward-logs/circleci-logs.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Set up a webhook trigger in CircleCI [#setup]
+
+Complete these steps to set up a CircleCI webhook to forward your logs to New Relic:
+
+1. Log in to [CircleCI](https://app.circleci.com/projects).
+
+2. Go to one of your CircleCI projects.
+
+3. Click **Project Settings**.
```
Comment (bradleycamacho) [inline src/content/docs/logs/forward-logs/circleci-logs.mdx:22]:
```suggestion
3. Click **Project settings**.
```

File: src/content/docs/logs/forward-logs/circleci-logs.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+4. In the sidebar of your Project Settings, click **Webhooks**.
+
+5. Click **Add Webhook**.
```
Comment (bradleycamacho) [inline src/content/docs/logs/forward-logs/circleci-logs.mdx:26]:
```suggestion
5. Click **Add webhook**.
```

File: src/content/docs/logs/forward-logs/circleci-logs.mdx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+6. Name your webhook.
+
+7. Enter New Relic's **Logs Endpoint**:
```
Comment (bradleycamacho) [inline src/content/docs/logs/forward-logs/circleci-logs.mdx:30]:
```suggestion
7. Enter New Relic's **Logs endpoint**:
```

File: src/content/docs/logs/forward-logs/circleci-logs.mdx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+   US: `"https://log-api.newrelic.com/log/v1?Api-Key=<NR_LICENSE_KEY>"`
+
+   EU: `"https://log-api.eu.newrelic.com/log/v1?Api-Key=<NR_LICENSE_KEY>"`
+
+   Use the region you set on your New Relic account. Replace `<NR_LICENSE_KEY>` with your New Relic license key, which you can [retrieve from the UI or API](/docs/apis/intro-apis/new-relic-api-keys/#manage-license-key).
+
+8. Select either **Workflow** or **Job**, depending on the type of event you want to trigger the logpush.
+
+9. If you've set up your receiving API or third-party service, click **Test Ping Event** to create a test event.
```
Comment (bradleycamacho) [inline src/content/docs/logs/forward-logs/circleci-logs.mdx:40]:
```suggestion
9. If you've set up your receiving API or third-party service, click **Test ping event** to create a test event.
```

**Review by bradleycamacho — DISMISSED** [review]
Looks good, just a few style edits

File: src/content/docs/logs/forward-logs/circleci-logs.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/logs/forward-logs/circleci-logs.mdx:26]: These are CircleCI UI things. I have to trust that they're capitalized correctly.

Comment (bradleycamacho) [reply src/content/docs/logs/forward-logs/circleci-logs.mdx:26]: Oh this is an interesting situation! Our style guide says we `use sentence case and bold for UI elements, even if the UI element is in a different case in the UI`, but maybe that should only apply to _our_ product and not when we reference other products?

**Comment by paperclypse** [comment]
@bradleycamacho I'm not going to commit your changes, because these are CircleCI UI procedures and I'm assuming that they're capitalized correctly.

**Comment by bradleycamacho** [comment]
@paperclypse happy to defer to you on this, just want to ensure you saw that one comment above!

> Oh this is an interesting situation! Our style guide says we `use sentence case and bold for UI elements, even if the UI element is in a different case in the UI`, but maybe that should only apply to _our_ product and not when we reference other products?

**Comment by paperclypse** [comment]
@bradleycamacho In this case, I think we want to prioritize trying to get this published today and then circle back later, if necessary.

**Comment by jcountsNR** [comment]
> @bradleycamacho In this case, I think we want to prioritize trying to get this published today and then circle back later, if necessary.

Just to add to this, the CircleCI doc link is [here](https://circleci.com/docs/2.0/webhooks/), I made the casing the same as the casing they used on their site. I don't have any insight as to whether our casing style should supersede theirs or not.

**Comment by paperclypse** [comment]
@jcountsNR I wanted to make sure we got this into the afternoon release. We can go back and edit the casing later, if need be.

**Pull request merged**